Сетка страниц и крупных блоков на флексах

MaryDeeJee opened the pull request:
---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/36/user/1826911/tasks/10)

:boom: https://htmlacademy-htmlcss.github.io/1826911-sedona-36/6/

MaryDeeJee committed:
Сетка страниц и крупных блоков на флексах

MaryDeeJee committed:
Delete index-background.jpg

MaryDeeJee committed:
добавила min-height карточкам отеля

Kartaviy-arteom commented on styles/styles.css:
```diff
@@ -23,6 +29,14 @@ body {
   background-color: #F2F2F2;
 }
 
+a {
+  text-decoration: none;
+}
+
+li {
+   list-style-type: none;
+}
+
```
посмотри этот критерий https://up.htmlacademy.ru/htmlcss/36/criteries#CSS-02

MaryDeeJee replied:
Сделала временно, для удобства просмотра
Когда будем проходить это, планирую поправить на более верный вариант

Kartaviy-arteom commented on styles/styles.css:
```diff
@@ -88,27 +171,27 @@ li {
-color: #FFFFFF;
-background-color: #7DB54F;
-border-radius: 10px;
+  font-style: normal;
```
font-family: "PT Sans"; нужно указывать семейство

Kartaviy-arteom commented on styles/styles.css:
```diff
@@ -166,17 +270,42 @@ text-transform: uppercase;
+.card {
+  width: 400px;
+  padding: 102px 85px;
+  box-sizing: border-box;
```
можно определить глобально, в большинстве случаев удобнее работать с border-box

Kartaviy-arteom commented on styles/styles.css:
```diff
@@ -207,13 +351,38 @@ text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  width: 1060px;
+  margin: 0 auto;
+  margin-bottom: 40px;
```
margin: 0 auto 40px - можно так объединить или так margin: 0 auto 40px auto; одно и тоже

Kartaviy-arteom commented on styles/styles.css:
```diff
@@ -238,6 +407,31 @@ text-transform: uppercase;
 }
 
+/* карточки отелей */
+.catalog__container {
+
+}
+
```
нет смысла писать пустые правила на будущее, скорее всего ты просто о них забудешь

MaryDeeJee replied:
как раз забыла убрать

Kartaviy-arteom commented on styles/styles.css:
```diff
@@ -238,6 +407,31 @@ text-transform: uppercase;
+
+}
+
+.catalog__list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0 70px;
+  margin-bottom: 20px;
```
justify-content: space-between; тебе не подойдет тут, посмотри как они себя будут вести, когда ты добавишь еще одну карточку

MaryDeeJee committed:
внесла правки

MaryDeeJee committed:
Исправила ошибки

- удалила font family, где не нужен
- добавила бордер бокс на весь документ
- сменила justify content у карточек отеля и добавила gap

MaryDeeJee committed:
Update styles.css
